lib/hyacinth/adapters/digital_object_search_adapter/solr/document_adapter.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Hyacinth
+  module Adapters
+    module DigitalObjectSearchAdapter
+      class Solr::DocumentAdapter
```

Comment:
This might be a weird and picky comment, but it feels like we are overusing the word "adapter". Because when i first saw the class name DocumentAdapter I thought it was at the same level with `DigitalObjectSearchAdapter` but its actually more of a helper class. Could we call it something more descriptive like Solr::DocumentCreation or something like that.